### ISSUE TITLE
About accuracy

### ISSUE THREAD

**Issue by feisichen** [dragen1860/MAML-Pytorch#73]
![ec2e2e5eb9e69b0cd678e1af2f1ffa33](https://user-images.githubusercontent.com/76902577/232542586-cbd0cf62-14da-49f5-81f4-e6a2d43ee643.png)

I trained the model using my own dataset.Does this mean that I have achieved 79.6% accuracy in few-shot classification? But I have only trained for ten minutes. . .I'm sure I didn't put the test set into the training set.

**Comment by sevenHsu** [comment]
Actually，your test set will be finetuned (train 10 steps for each support set) after every 500 steps training on training set. So you can see the acc growth from 0,2615 to 0.796 via 10 steps finetune.

**Comment by feisichen** [comment]
Thank you. Are the support and query sets automatically split from the test set?

**Comment by sevenHsu** [comment]
They were randomly selected from each task test dataset.

**Comment by LittleShuo** [comment]
Hello, where did you get those csv files in your miniimagenet dataset?

**Comment by feisichen** [comment]
> Hello, where did you get those csv files in your miniimagenet dataset?

Sorry, I didn't use the miniimagenet dataset, so I'm not very familiar with it. I'm using my own dataset, where the images for each category are placed in their respective folders. So, I wrote a simple Python program to generate the csv files.

**Comment by LittleShuo** [comment]
Ok, thank you very much for your reply.
